aiohttp/wsgi.py

```diff
@@ -105,7 +105,7 @@ def create_wsgi_environ(self, message, payload):
         if script_name:
             path_info = path_info.split(script_name, 1)[-1]
 
-        environ['PATH_INFO'] = path_info
+        environ['PATH_INFO'] = unquote(path_info).encode().decode('latin-1')
```

Comment:
@fafhrd91 used utf-8 in decoding response headers for client (266b38625476f4f86e7111dc05783ccbe6253a1)
Maybe we need to make modifications in parser, not in wsgi?